.travis.yml

```diff
@@ -51,3 +53,39 @@ jobs:
     - <<: *buildtag
       env:
         - DOCKERFILE=Dockerfile.alpine TARGET=rel TAG_SUFFIX="-alpine"
+
+    - &helmlint
+      name: Helm lint
+      stage: test
+      env:
+        - KUBECTL_VERSION=v1.11.3
+        - HELM_VERSION=v2.11.0
+      install:
+        - wget https://storage.googleapis.com/kubernetes-release/release/${KUBECTL_VERSION}/bin/linux/amd64/kubectl -O /tmp/kubectl
+        - chmod +x /tmp/kubectl
+        - wget http://storage.googleapis.com/kubernetes-helm/helm-${HELM_VERSION}-linux-amd64.tar.gz -O /tmp/helm.tar.gz
+        - tar xzf /tmp/helm.tar.gz -C /tmp --strip-components=1
+        - chmod +x /tmp/helm
+        - export PATH=$PATH:/tmp
+      before_script:
+        - helm init --client-only
+      script:
+        - helm lint helm/vernemq
+        - helm template -f helm/vernemq/values.yaml helm/vernemq
+        - helm package --save=false helm/vernemq
+
+    - <<: *helmlint
+      name: Helm deploy
+      stage: deploy-helm-chart
+      after_script:
+        - git config user.name "travis"
+        - git config user.email "deploy@travis-ci.org"
+        - git stash push -u -- vernemq-*.tgz
+        - git config --replace-all remote.origin.fetch +refs/heads/*:refs/remotes/origin/*
+        - git fetch origin helm-charts
+        - git checkout helm-charts
+        - git stash pop
+        - helm repo index .
+        - git add vernemq-*.tgz index.yaml
+        - git commit -m 'Add helm chart release $TRAVIS_BRANCH'
+        - git push origin helm-charts
```

Comment:
`git push -q https://$GITHUB_USER:$GITHUB_TOKEN@github.com/vernemq/docker-vernemq helm-charts &>/dev/null`

Reply:
Ì've already configured GITHUB_USER and GITHUB_TOKEN as environment variables for the Travis project. So this should all we need.